Bottom: 7f2ba7d9327fc394f010914430a502aa9c1221d6
Top:    9c8c886a489050f572182efc391e495116753f62
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-13 10:57:44 -0500

Use internal email server

Use our internal email server instead of an external relay


---
diff --git a/ansible-playbook/group_vars/all/credentials.yml b/ansible-playbook/group_vars/all/credentials.yml
index da5b9b71..78b584d2 100644
--- a/ansible-playbook/group_vars/all/credentials.yml
+++ b/ansible-playbook/group_vars/all/credentials.yml
@@ -1,6 +1,7 @@
 ---
 # passwords
 buildbot_admin_password:             '{{ lookup("password", playbook_dir + "/credentials/buildbot/admin"                      , length=20) }}'
+buildbot_email_password:             '{{ lookup("password", playbook_dir + "/credentials/buildbot/buildbot@novatech-llc.com"  , length=20) }}'
 buildbot_worker_ntel_password:       '{{ lookup("password", playbook_dir + "/credentials/buildbot/worker_ntel"                , length=20) }}'
 buildbot_worker_ptxdist_password:    '{{ lookup("password", playbook_dir + "/credentials/buildbot/worker_ptxdist"             , length=20) }}'
 buildbot_webhook_secret:             '{{ lookup("password", playbook_dir + "/credentials/buildbot/webhook"                    , length=20) }}'
diff --git a/ansible-playbook/group_vars/all/email.yml b/ansible-playbook/group_vars/all/email.yml
new file mode 100644
index 00000000..2e5c0c22
--- /dev/null
+++ b/ansible-playbook/group_vars/all/email.yml
@@ -0,0 +1,4 @@
+email_accounts:
+  buildbot:
+    user: buildbot
+    password: '{{ buildbot_email_password }}'
